_schemaV2.graphql

```diff
@@ -2474,7 +2476,10 @@ type Artwork implements Node & Searchable & Sellable {
   framed: ArtworkInfoRow
     @deprecated(reason: "Consider using isFramed field (boolean) instead")
   framedDepth: String
+  framedDiameter: Float
```

Comment:
I'm confused by why this `framedDiameter` is Float and the above `framedDepth` is String? They both defined as Floats in gravity. We should probably keep it consistent one way or the other?